ui/appui-react/src/appui-react/configurableui/ConfigurableUiContent.tsx

```diff
@@ -73,6 +73,8 @@ export interface ConfigurableUiContentProps extends CommonProps {
   widgetOpacity?: number;
   /** Controls if the widget icons should be displayed. Uses redux store as a fallback. Defaults to `true`. */
   widgetIcon?: boolean;
+  /** Controls if the active widget's label should always be displayed. Defaults to `false`. */
+  activeWidgetLabel?: boolean;
```

Comment:
Let's rename to `displayActiveWidgetLabel`/`showActiveWidgetLabel` to indicate a `boolean` (even if this is consistent with poorly named `widgetIcon` above)